docs/index.asciidoc

```diff
@@ -21,9 +21,10 @@ include::{include_path}/plugin_header.asciidoc[]
 
 ==== Description
 
-The Elastic EnterpriseSearch Integration Plugin provides integrated plugins for working with
-AppSearch service:
+The Elastic Enterprise Search integration plugin provides integrated plugins for working with
+App Search and Workplace Search services:
 
-- {logstash-ref}/plugins-outputs-elastic_app_search.html[AppSearch Output Plugin]
+- {logstash-ref}/plugins-outputs-elastic_app_search.html[App Search output plugin]
+- {logstash-ref}/plugins-outputs-elastic_workplace_search.html[Workplace Search output plugin]
```

Comment:
```suggestion
- Workplace Search output plugin
```

Reply:
The generated file doesn't exist yet, and linking to it will break the build. Let's hard code it for now, and I will add back the link when I hook up the other doc pieces.

Reply:
Should I make this change before merging?

Reply:
It won't break the build until after we publish the gem, but I'm always hesitant to merge time bombs. If this is the only thing preventing you from merging, please make this change and I'll add your link back as part of a PR I have inflight now.

Reply:
Removed!